configure-load-base-split.md

```diff
@@ -23,15 +23,21 @@ Previously, the solution to this problem was to manually execute a command to sp
 
 ## Implementation principles
 
-Load Base Split automatically splits the Region based on statistics. It identifies the Regions whose read load consistently exceeds the threshold for 10 seconds, and splits these Regions at a proper position. When choosing the split position, Load Base Split tries to balance the access load of both Regions after the split and avoid access across Regions.
+Load Base Split automatically splits the Region based on statistics. It identifies the Regions whose read load or CUP usage consistently exceeds the threshold for 10 seconds, and splits these Regions at a proper position. When choosing the split position, Load Base Split tries to balance the access load of both Regions after the split and avoid access across Regions.
 
 The Region split by Load Base Split will not be merged quickly. On the one hand, PD's `MergeChecker` skips the hot Regions; on the other hand, PD also determines whether to merge two Regions according to `QPS` in the heartbeat information, to avoid the merging of two Regions with high `QPS`.
 
 ## Usage
 
-The Load Base Split feature is currently controlled by the `split.qps-threshold` parameter (QPS threshold) and `split.byte-threshold` parameter (traffic threshold). If the sum of all types of read requests per second for a Region exceeds the QPS threshold or traffic threshold for 10 consecutive seconds, PD splits the Region.
+The Load Base Split feature is currently controlled by three parameters:
 
-Load Base Split is enabled by default, but the parameter is set to a rather high value. `split.qps-threshold` defaults to `3000` and `split.byte-threshold` defaults to 30MB/s. If you want to disable this feature, set the two thresholds high enough at the same time.
+- `split.qps-threshold`: The threshold of QPS for the Region to be split, default is `3000` per second.
+- `split.byte-threshold`: The threshold of read load for the Region to be split, default is 30 MiB per second.
+- `split.region-cpu-overload-threshold-ratio`: The threshold of CPU usage (the percentage of CPU time of the read thread pool) for the Region to be split, default is `0.25`.
+
+If the sum of all types of read requests per second for a Region exceeds the QPS threshold or traffic threshold for 10 consecutive seconds, PD splits the Region.
```

Comment:
What about the CPU usage threshold?

Reply:
Also, it's not that "PD splits the Region", this is a TiKV feature that doesn't involve PD, so the precise description should be like "TiKV tries to split the Region".